### PR TITLE
Dwell voltage correction

### PR DESCRIPTION
**Pull request by mck1117**
#3099

### CODE REVIEW

**Comment by rusefillc** [comment]
please mention on CHANGELOG

File: firmware/tunerstudio/rusefi.input [hunk 1]
```diff
@@ -604,6 +604,14 @@ enable2ndByteCanID = false
...
       yBins       = sparkDwellValues
       gauge       = RPMGauge
+
+	curve = dwellVoltageCorrection, "Dwell voltage correction"
+      columnLabel = "Battery Volts", "Dwell"
+      xAxis       =  0, 8000, 9
+      yAxis       =  6, 16, 5
+      xBins       = dwellVoltageCorrVoltBins, VBatt
+      yBins       = dwellVoltageCorrValues
+      gauge       = VBattGauge
```
[the remark this answers is not in the document]
Comment (mck1117) [reply firmware/tunerstudio/rusefi.input:614]: can't do multiple gauges :(

[the remark this answers is not in the document]
Comment (dron0gus) [reply firmware/tunerstudio/rusefi.input:614]: gauge1, gauge2 is not supported in table editing dialogs?

**Review by dron0gus — APPROVED** [review]

File: firmware/tunerstudio/rusefi.input (hunk 1 above)
[the remark this answers is not in the document]
Comment (dron0gus) [reply firmware/tunerstudio/rusefi.input:614]: http://www.tunerstudio.com/index.php/manuals/80-command-buttons

**Pull request merged**